docs-src/changelog.md

```diff
@@ -2,6 +2,13 @@
 HypoFuzz uses [calendar-based versioning](https://calver.org/), with a
 `YY-MM-patch` format.
 
+## 24.12.1
```

Comment:
😅

Reply:
I'll assume this is a poke to redate before merging! (and eventually set up some CI to do this for us)